app/AccountancyModule/EducationModule/templates/Default/@eventsGrid.latte

```diff
@@ -1,5 +1,10 @@
 {extends $baseTemplate}
 
 {block filters}
+    {include #filter-type-search}
     {include #filter-type-select name => 'year', label => 'Rok', icon => 'fas fa-calendar'}
 {/block}
+
+{define col-displayName}
+    <a href="{plink Education:default $item->getId()}">{$item->getName()}</a>
+{/define}
```

Comment:
Hele tady moc nechápu, proč tohle funguje jinak, než to samý pro tábory - když jsem to zkoušel stejně, tak to spadlo...

https://github.com/skaut/Skautske-hospodareni/blob/3dad5c883ee599d7f434540054b4851f2036c7cd/app/AccountancyModule/CampModule/templates/%40campsGrid.latte#L22-L24

Reply:
Protože u táborů to odkazuje na sloupec `displayName`, ale uz se tam používáme `name` a rovnou se z toho dělá link viz https://github.com/skaut/Skautske-hospodareni/blob/master/app/AccountancyModule/CampModule/presenters/DefaultPresenter.php#L34-L35 . Tak bych to udělal taky v presenteru už

Tady ten nepouzitý kod odeberu https://github.com/skaut/Skautske-hospodareni/pull/2358

Reply:
Aha, takže to bere přímo z toho presenteru, to mi doteď nesecvaklo... Díky!